kinto_webpush/__init__.py

```diff
@@ -12,3 +14,8 @@ def includeme(config):
     docs = "https://github.com/Kinto/kinto-webpush"
     config.add_api_capability("webpush", message, docs, version=__version__)
     config.scan('kinto_webpush.views')
+
+    # Listen to every resources
+    config.add_subscriber(on_resource_changed, ResourceChanged,
+                          for_resources=('bucket', 'group',
+                                         'collection', 'record'))
```

Comment:
Note: If you don't specify the resources, notifications will come for every of them